frontend/public/features.ts

```diff
@@ -44,6 +45,7 @@ export enum FLAGS {
   CAN_LIST_PV = 'CAN_LIST_PV',
   CAN_LIST_STORE = 'CAN_LIST_STORE',
   CAN_LIST_CRD = 'CAN_LIST_CRD',
+  CAN_PROMOTE_CRD = 'CAN_PROMOTE_CRD',
```

Comment:
CAN_CREATE_CONSOLE_EXTENSION

I think "promote" in general is not the right terminology, so I'd try to get away from that in names.